### PR TITLE
test: allow either order for tooManyCorruptedSource_invalidTest

### PR DESCRIPTION
**Pull request by ed-g**
**Summary:**

tooManyCorruptedSource_invalidTest for https://github.com/MobilityData/gtfs-validator/pull/1086#issuecomment-1026223882

### CODE REVIEW

File: output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java [hunk 1]
```diff
@@ -347,15 +349,20 @@ public void tooManyCorruptedSource_invalidTest() throws Exception {
...
-                    ACCEPTANCE_TEST_REPORT_FOLDER_NAME,
-                    SOURCES_CORRUPTION_REPORT_JSON)))
-        .isEqualTo(
+    MatcherAssert.assertThat(
+        retrieveReportString(
+          resolve(
+            NEW_NOTICES_TYPE_FOLDER_NAME,
+            ACCEPTANCE_TEST_REPORT_FOLDER_NAME,
+            SOURCES_CORRUPTION_REPORT_JSON)),
+        Matchers.anyOf(
```
Comment (maximearmstrong) [inline output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java:358]: I'm okay with the code. Did you encountered a use case where the order of `source-id-5` and `source-id-4` is reversed? I tried to run the original test after reversing the order and the test is failing.

**Review by maximearmstrong — COMMENTED** [review]
Thank you for your contribution @ed-g! I have a question in-line before approving this PR. 
If you have a use case where the order of the sources is reversed, could you please provide more information on it?

File: output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java (hunk 1 above)
Comment (maximearmstrong) [reply output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/MainTest.java:358]: Oh! I just realized your explanation was in #1086. Sorry about that. I am unable to reproduce the bug. Could you provide before and after screenshots in this PR to document the fix?

**Comment by maximearmstrong** [comment]
Also, the acceptance tests on PRs from forks were fixed with #1102. Could you fix the format of the code and use the acceptance tests workflow from the master branch? Then we can see if all the checks pass :)

**Comment by ed-g** [comment]
The only time I've seen an reversed order is in January when I was doing initial tests.

If you're not seeing it we may not need to add the case to the test.

**Comment by maximearmstrong** [comment]
I tested this again with different Java versions and I didn't experience the issue. If it happens again, please let us know :)

**Pull request closed without merging**